docusaurus.config.ts

```diff
@@ -69,6 +70,12 @@ const config: Config = {
   ],
 
   themeConfig: {
+    algolia: {
+      appId: 'H6BESWXMON',
+      apiKey: '94b54b551433d306dade3f9cdb84b92b',
```

Comment:
Hope these are not secrets.

Reply:
These are public API keys, safe to use in frontend.